src/NetMQ/NetMQ.nuspec

```diff
@@ -16,9 +16,6 @@
     </releaseNotes>
     <copyright>Copyright 2016</copyright>
     <tags>ZeroMQ 0MQ CLRZMQ NetMQ Messaging ZMQ transport distributed</tags>
-    <dependencies>
-      <dependency id="AsyncIO" version="0.1.18.0" />
```

Comment:
I did not. I made sure nuget will properly use the config in the project. AsyncIO becomes a dependency because it is in the config.